### PR TITLE
Treat empty worker output the same as empty object '{}'

### PR DESCRIPTION
**Pull request by xavi-**
cc @mohit

### CODE REVIEW

**Comment by mohit** [comment]
💯

File: cmd/sfncli/test_scripts/stdout_empty_output.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+echo ""
```
Comment (rgarcia) [inline cmd/sfncli/test_scripts/stdout_empty_output.sh]: in reality the workers we're talking about don't even echo a newline, so I'd consider removing this

Comment (xavi-) [reply cmd/sfncli/test_scripts/stdout_empty_output.sh]: Good point.  Updated in c3a79bf1920c29877d768ec2233e14445d55aaa0

**Comment by mohit** [comment]
bumping version on master to release this.

**Comment by mohit** [comment]
cc: @ahsojar

**Comment by xavi-** [comment]
oof, sorry about that

**Pull request merged**